src/main.js

```diff
@@ -75,20 +84,28 @@ function createRandomCover() {
 function showForm() {
```

Comment:
This comment refers to lines 76 - 80 (I wasn't able to comment there since it wasn't changed): 
These 4 lines should be broken out into their own function. If you think about it, lines 71 - 75 are changing/using the **data model**, accessing data to create a new cover instance, while lines 76 - 80 are all DOM-related, using those values to display the new cover. In general, you always want to separate out functions that update/access data from those that are just for DOM manipulation. 

Additionally, line 76 - 80 are exactly the same as lines 114 - 117, so if this lives in a new displayCover function, you'll be able to re-use it there and DRY up your code a bit more. Lmk if you have any questions about this!